src/arrayOfPeople.js

```diff
@@ -38,7 +38,11 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  const peopleArr = [];
+  people.forEach(person => peopleArr.includes(person) ? 
+                peopleArr.splice(peopleArr.indexOf(person), 1) :
```

Comment:
Will be better to store the result of indexOf in variable and use it where you call includes for better performance